frontend/src/views/NegotiationView.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) 2024 Volkswagen AG
+Copyright (c) 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST)
+Copyright (c) 2024 Contributors to the Eclipse Foundation
+
+See the NOTICE file(s) distributed with this work for additional
+information regarding copyright ownership.
+
+This program and the accompanying materials are made available under the
+terms of the Apache License, Version 2.0 which is available at
+https://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import { useNegotiations } from '@hooks/edc/useNegotiations';
+
+export const NegotiationView = () => {
+    const { negotiations  } = useNegotiations();
+    return (
```

Comment:
Negotiations are not differentiated into cards or something:

![image](https://github.com/eclipse-tractusx/puris/assets/116013795/c1e83465-c2da-4d1e-a89c-47077f3e5f74)